hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconNodeManager.java

```diff
@@ -44,6 +40,10 @@
 import org.apache.hadoop.ozone.protocol.commands.SCMCommand;
 import org.apache.hadoop.ozone.recon.ReconUtils;
 import org.apache.hadoop.util.Time;
+
+import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_DB_CACHE_SIZE_DEFAULT;
+import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_DB_CACHE_SIZE_MB;
+import static org.apache.hadoop.ozone.recon.ReconConstants.RECON_SCM_NODE_DB;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
```

Comment:
ReconNodeManager also uses the MetadataStore to persist Node information. Any reason why we don't want to change this also like the rest?

Reply:
Not particular reason, just reducing the size the patch. I renamed my original Jira to fix SCM only. Recon and Datanode can be fixed in next two jira. 

(But thanks to point it to me, I was not aware of this. But it also can be moved to SCM. AFAK @Nanda had a plan to persist it on the SCM side, too...)

Reply:
Sounds good, thanks!